apps/modernization-api/src/main/java/gov/cdc/nbs/controller/RedirectController.java

```diff
@@ -70,6 +71,19 @@ public RedirectView redirectAdvancedSearch(
       HttpServletResponse response) {
     return redirectionService.handleRedirect(ADVANCED_SEARCH, request, response);
   }
+  
+  /**
+   * Intercepts patient-profile from legacy page and re-routes/forwards the patient-profile to the modernized 
+   * patient profile page
+   */
+  @ApiIgnore
+  @PostMapping("/nbs/redirect/patientProfile")
+  public RedirectView redirectPatientProfile(
+          @RequestParam String MPRUid, 
+          HttpServletRequest request,
+          HttpServletResponse response) {
```

Comment:
There is a code smell wanting to rename MPRUid to match standard variable naming conventions.

could change to `@RequestParam(name = "MPRUid") String patientId` or something similar